README.md

```diff
@@ -41,10 +41,28 @@ in Internet Explorer on Windows 7 for instance (because of the wrong MIME type).
 
 JavaScript Cookie supports [npm](https://www.npmjs.com/package/js-cookie) and [Bower](http://bower.io/search/?q=js-cookie) under the name `js-cookie`.
 
+- #### Bower
+  ```
+  bower install js-cookie --save
+  ```
+
+- #### NPM
+  ```
+  npm install js-cookie --save
+  ```
+
 ### Module Loaders
 
 JavaScript Cookie can also be loaded as an AMD, CommonJS or [ES6](https://github.com/js-cookie/js-cookie/issues/233#issuecomment-233187386) module.
 
+```javascript
+import Cookie from 'js-cookie';
+```
+
+```javascript
+const Cookie = require('js-cookie');
```

Comment:
The link above sends the reader to https://github.com/js-cookie/js-cookie/issues/233#issuecomment-233187386. There there's a discussion on how to use it using the current mainstream tools.

Maybe we can document the ES6 module version because it's standard. But using code snippets for AMD might not be optimal since it occupies space in the README and it's a not standard thing.

We need to take care not to clutter the README with a lot of stuff. Maybe some of these could be in the Wiki?